pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -783,12 +784,28 @@ func (br *OVSBridge) GetOVSVersion() (string, Error) {
 		klog.Error("Transaction failed: ", err)
 		return "", NewTransactionError(err, temporary)
 	}
+
 	if len(res[0].Rows) == 0 {
-		klog.Warning("Could not find ovs_version")
-		return "", nil
+		klog.Warning("Could not find ovs_version in the OVS query result")
+		return "", NewTransactionError(fmt.Errorf("no results from OVS query"), false)
+	} else {
+		return ParseOvsVersion(res[0].Rows)
 	}
+}
 
-	return res[0].Rows[0].(map[string]interface{})["ovs_version"].(string), nil
+func ParseOvsVersion(ovsReturnRow interface{}) (string, Error) {
```

Comment:
Make the function private since it's not used out of the package?

Reply:
id say its a public function bc its stateless and nothing wrong w/ others using it... but sure can make private for now